exercises/practice/rna-transcription/.meta/config.json

```
@@ -1,6 +1,15 @@
 {
   "blurb": "Given a DNA strand, return its RNA Complement Transcription.",
-  "authors": [],
+  "authors": [
+    "SuperPaintman"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "jackhughesweb",
+    "kytrinyx",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

I suggest adding my name since I did revisions to the exercise that resulted in either adding/removing test cases or modifying them due to syncing the exercise to the spec.

```suggestion
  "contributors": [
    "Stargator"
  ],
```